.travis.yml

```diff
@@ -33,12 +37,12 @@ after_success:
 - bash <(curl -s https://codecov.io/bash)
 deploy:
   - provider: script
-    script: make deploy
+    script: make -jO deploy
```

Comment:
What does "-j0" do?
After reading the following excerpt from the GNU make 4 manual the effect of the value 0 wasn't completely clear to me:

> If the ‘-j’ option is followed by an integer, this is the number of recipes to execute at
> once; this is called the number of job slots. If there is nothing looking like an integer after
> the ‘-j’ option, there is no limit on the number of job slots. The default number of job
> slots is one, which means serial execution (one thing at a time).